test/e2e/End2EndSpec.scala

```diff
@@ -39,7 +37,7 @@ class End2EndSpec (arguments: Arguments) extends Specification with GuiceFakeApp
 
   }
 
-  private def runWebdriverTests = "yarn test-e2e".run().exitValue()
+  private def runWebdriverTests = "yarn test-e2e -C 1".run().exitValue()
```

Comment:
I think, it would be better to incorporate this change in the `test-e2e` entry in the package.json. That way, manual executions of this command will also use the concurrency flag. Do you agree?

Reply:
Yes I agree. Not sure where to pass it there, though. Should it be in `test.sh`? Or its call in package.json?

Reply:
Good point, it's probably best to have it in `test.sh` where `*.e2e.js` is passed (line 48).